src/validation.h

```diff
@@ -766,7 +766,7 @@ class CChainState
         size_t max_mempool_size_bytes) EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
 
     /** Return list of MN EHF signals for current Tip() */
-    std::unordered_map<uint8_t, int> GetMNHFSignalsStage(const CBlockIndex* pindex) EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
```

Comment:
is there anywhere we acquired cs_main where we can drop it now?

Reply:
already done: https://github.com/dashpay/dash/pull/5753/files/dcae438aa1fb1be64376c8d14b08f928d5e65f71#diff-6bf9d2e37d503f2f3381cd3a41c26e0b9670f26c2cd4571e98d2af9e3767c1a1L399

Reply:
GitHub can't find this?

Reply:
link become invalid with every push.

https://github.com/knst/dash/commit/0fc09466bd712b507ee38cea5848333a5e3263f2 - this should be available now